config-vm.md

```diff
@@ -61,8 +61,54 @@ This image contains the root filesystem that the virtual machine **`kernel`** wi
     }
 ```
 
+## <a name="HwConfigObject" /> HWConfig Object
+
+**`hwConfig`** (object OPTIONAL) Specifies the hardware configuration that should be passed to the VM.
+* **`deviceTree`** (string OPTIONAL) Path to the container device-tree file that should be passed to the VM.
+* **`vcpus`** (int OPTIONAL) Number of virtual cpus for the VM.
+* **`memKB`** (int OPTIONAL) Maximum memory in KB allocated to the VM.
```

Comment:
Is there a reason why it is in KB units?

Reply:
To make this specification usable to start small real time VMs which may require < 1MB memory on the small systems. Providing memory in bytes is excessive from my standpoint.

Reply:
We have been using bytes in other places, so using bytes is preferable.

Implementations may still reject a config when `bytes % 1024 != 0`.

Reply:
Renamed parameter to "memory" and provided it in bytes.

Reply:
@oleksiimoisieiev Thanks 👍